pupa/scrape/jurisdiction.py

```diff
@@ -42,9 +42,6 @@ def as_dict(self):
                 'legislative_sessions': self.legislative_sessions,
                 'feature_flags': self.feature_flags, 'extras': self.extras, }
 
-    def get_session_list(self):
```

Comment:
No, it should have been part of https://github.com/opencivicdata/pupa/pull/243. I can put into a separate PR, if you want.